packit_service/worker/build/copr_build.py

```diff
@@ -254,25 +256,25 @@ def test_target2build_target(self, test_target: str) -> str:
                 targets:
                       fedora-35-x86_64
 
+
```

Comment:
> Tricky question in the end: is it a valid use case to test more builds with one test target? This would not work in that case correctly.

But they can still manually configure that themself, can't they?

Reply:
hmm, I don't think it is possible by the proper way, this question came to my mind looking into [this](https://github.com/sgallagher/sscg/blob/b9f757736f73db8c58bb9e422e018ab84eabd51f/.packit.yaml#L19) config, where there is build/test both for `epel-8` and `centos-stream-8` and `epel-8` will be in the end tested as well in `centos-stream-8` (I am not sure whether this is intended by the users or they just don't know about default mapping)...and the `test_target2build_target(centos-stream-8)` would return only on of the build targets, since we always talked only about 1 build: many tests and not the other way

Reply:
The original ask for mappings was to allow reusing one build (mainly centos) in multiple test targets and avoiding a need to have all the centos clones like Alma or Rocky in Copr.

Regarding that specific config, there is no `epel` target so it needs to use something else like `centos-stream` or `rhel`.
And by this change, we will not change anything for them, will we?

Reply:
This change will change the behaviour for them, since `test_target2build_target(centos-stream-8)` will return always only one matching build target (first in the list).